src/routes/RouteResolver.js

```
@@ -0,0 +1,25 @@
+const pagesData = {
+  AUTH: { route: '/auth' },
+  HOME: { route: '/' },
+  COMPANIES: { route: '/companies' },
+  CATEGORIES: { route: '/categories' },
+  CATEGORY: { route: '/category' },
+  'FORGOT-PASSWORD': { route: '/forgot-password' },
+  SERVICES: { route: '/services' },
+  SERVICE: { route: '/service' },
+  'SIGN-IN': { route: '/sign-in' },
+  'SIGN-UP': { route: '/sign-up' },
+  'SIGN-OUT': { route: '/sign-out' },
+  USER: { route: '/user' },
+};
+
+const getPageUrl = pageCode => {
+  const pageInfo = pagesData[pageCode];
+  if (!pageInfo) {
+    console.warn('No page with code ' + pageCode);
```

I would throw an error here to indicate to developers that app will not work as expected

---

Yuo mean to use a console.error?

---

yup